test/units/architecture/architect.test.js

```diff
@@ -1,22 +1,34 @@
 let { assert, expect } = require('chai');
 let should = require('chai').should();
-let carrot = require('../../../src/carrot');
+let {architect, Network, Node } = require('../../../src/carrot');
```

Comment:
We can start turning all of these imports in to `const`s they should not be overwritten throughout testing and we can save on the memory.